package.json

```diff
@@ -217,4 +217,4 @@
         "use-debounce": "^7.0.1",
         "use-file-input": "^1.0.0"
     }
-}
+}
```

Comment:
I feel like this keeps appearing. Any ideas why? No need for this change

Reply:
after latest issues I always keepa copy of package.json and yarn from main and past them in my env. then push and commit ,maybe that is the cause of this line